### PR TITLE
Feature/theme

### PR DESCRIPTION
**Pull request by MayuRuru**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+import 'package:flex_color_scheme/flex_color_scheme.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:hive_flutter/hive_flutter.dart';
 
 Future main() async {
   runZonedGuarded<Future<void>>(() async {
     WidgetsFlutterBinding.ensureInitialized();
-    // Hive-specific initialization
+    /// Hive-specific initialization
```
Comment (selankon) [inline lib/main.dart]: Inline comments are with only two `//`

File: lib/main.dart [hunk 2]
```diff
@@ -21,22 +21,80 @@ Future main() async {
...
+        ),
+        visualDensity: FlexColorScheme.comfortablePlatformDensity,
+        textTheme: buildTextTheme(),
+      ),
     );
   }
-}
\ No newline at end of file
+
+  TextTheme buildTextTheme() {
```
Comment (selankon) [inline lib/main.dart:62]: Should this be in a separated file?

File: lib/src/core/theme/app_theme.dart [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+
+//final ThemeData empriusTheme = _empriusTheme();
```
Comment (selankon) [inline lib/src/core/theme/app_theme.dart:3]: Why this commented code? Commented code MA-LO

File: lib/src/core/theme/theme_controller.dart [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import 'package:flex_color_scheme/flex_color_scheme.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+final lightTheme = FlexColorScheme.light(scheme: FlexScheme.verdunHemlock).toTheme;
+final darkTheme = FlexColorScheme.dark(scheme: FlexScheme.verdunHemlock).toTheme;
+
+final themeProvider = StateNotifierProvider<ThemeNotifier, ThemeData>((ref) {
```
Comment (selankon) [inline lib/src/core/theme/theme_controller.dart:8]: BRAVO, good job

Comment (selankon) [reply lib/src/core/theme/theme_controller.dart:8]: This is just a comment nothing to review

**Review by selankon — CHANGES_REQUESTED** [review]

File: lib/src/core/theme/app_theme.dart (hunk 3 above)
Comment (MayuRuru) [reply lib/src/core/theme/app_theme.dart:3]: I tried to translate the automated colorScheme dependency to my own custom theme to have more control but didn't manage to use it properly - so I kept with the dependency but saved this file to have access to the color code when needed.

**Comment by MayuRuru** [comment]
Solved and merged

**Pull request merged**